jupyter-lab/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM jupyter/minimal-notebook:latest
+LABEL org.opencontainers.image.source https://github.com/scilifelabdatacentre/stackn-images
+
+USER root
+RUN /bin/bash -c "apt update"
+RUN /bin/bash -c "apt install curl -y"
```

Comment:
Nice. We can reduce the number of container layers in this file by updating for example:
```code
RUN /bin/bash -c "apt update"
RUN /bin/bash -c "apt install curl -y"
```
to 
```code
RUN /bin/bash -c "apt update && \
    /bin/bash -c "apt install curl -y"
```
and i think we can reduce 
```code
RUN pip uninstall minio -y
RUN pip install minio==7.0.2
```
to
```code
RUN  pip install --force-reinstall "minio==7.0.2"
```

I'll mark this as `review` but change what you believe is necessary.

Reply:
Good. I will make the change.